gma/src/android/native_ad_image_android.cc

```diff
@@ -143,5 +148,42 @@ double NativeAdImage::scale() const {
   return internal_->scale;
 }
 
+/// Gets the auto loaded image as a vector of bytes.
+const std::vector<unsigned char> NativeAdImage::image() const {
+  JNIEnv* env = ::firebase::gma::GetJNI();
```

Comment:
Similar comment to the iOS implementation (below) regarding this method blocking.

Reply:
Thanks David! Updated it to return a future and process the image loading asynchronously, rather than as a blocking call in iOS implementation as well.